MovieApp/MovieApp/App/AppDelegate.swift

```diff
@@ -4,15 +4,18 @@ import CoreData
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    func application(_ application: UIApplication,
```

Comment:
Please read our code style and check how to format functions with multiple arguments.